package.json

```diff
@@ -47,7 +47,8 @@
     "typescript": "^5.3.3",
     "viem": "^2.20.1",
     "vite": "^5.2.13",
-    "vitest": "^1.6.0"
+    "vitest": "^1.6.0",
+    "ethereum-cryptography": "2.2.1"
```

Comment:
what's this dep in use for?

Reply:
I can delete - it's not anymore. I was doing the signing directly at one point I'll remove 🧹